test/integration/job/job_test.go

```diff
@@ -1775,6 +1865,12 @@ func getCompletionIndex(p *v1.Pod) (int, error) {
 	return strconv.Atoi(v)
 }
 
+func failPod(ctx context.Context, t *testing.T, clientSet clientset.Interface, jobObj *batchv1.Job) {
+	if err, _ := setJobPodsPhase(ctx, clientSet, jobObj, v1.PodFailed, 1); err != nil {
```

Comment:
maybe just inlining these lines is not too bad. Otherwise, worth calling t.Helper()